src/lib.rs

```diff
@@ -663,6 +663,14 @@ pub mod pallet {
 				BoundedVec::<_, T::MaxInvulnerables>::try_from(new_with_keys)
 					.map_err(|_| Error::<T>::TooManyInvulnerables)?;
 
+			// Make sure we meet the minimum eligible collator requirement.
```

Comment:
Make sure the minimum eligible collator requirement is met.

Reply:
Changed.